multiplex.json

```diff
@@ -24,6 +24,11 @@
             "args": [ "wsize", "rsize", "nthreads", "duration" ],
             "vals": "[1-9][0-9]*"
         },
+        "positive_float": {
+            "description": "a floating point number greater than 0",
+            "args": [ "think" ],
+            "vals": "[0-9*\\.[0-9]*"
```

Comment:
Is this right? Or should it have been `^[0-9]*\.[0-9]*$`?

Reply:
A single backslash actually does not work.  I think this is because of some conversion going on.

Reply:
Ok, that sorta rings a bell.  But shouldn't that opening bracket be closed?